src/WidgetBase.ts

```diff
@@ -320,6 +323,19 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> extends E
 				_registries.add(newProperties.defaultRegistry, true);
 			}
 		}
+		this._defineRegistryItems();
+	}
+
+	private _defineRegistryItems(): void {
+		const registryItems: DefineDecoratorConfig[] = this.getDecorator('registryItem');
+		const { defaultRegistry } = this._registries;
+		for (let i = 0; i < registryItems.length; i++) {
+			const { label, item } = registryItems[i];
+			if (defaultRegistry && defaultRegistry.has(label) === false) {
```

Comment:
Any particular reason for `defaultRegistry.has(label) === false` instead of `!defaultRegistry.has(label)`?  Not that I have any particular concern, but `has` returns a boolean, so it is unnecessary to guard against falsey values and there is a very minor performance difference:

<img width="968" alt="strict_comparison_ _jsperf" src="https://user-images.githubusercontent.com/1282577/29970434-741adc78-8f1c-11e7-82a9-ddac0ca8a16b.png">

Reply:
Just a habit, I can change. (just curious is that true across all the browsers?)

Reply:
https://jsperf.com/strict-comparison-kpk/1

You gotta love JVM engines though
<img width="972" alt="dashboard" src="https://user-images.githubusercontent.com/1282577/29971239-79aeae46-8f1f-11e7-9da7-e126d1cda41a.png">
:

<img width="971" alt="strict_comparison_ _jsperf" src="https://user-images.githubusercontent.com/1282577/29971151-27d56880-8f1f-11e7-8f2a-b1c34f442632.png">
<img width="964" alt="strict_comparison_ _jsperf" src="https://user-images.githubusercontent.com/1282577/29971165-2e30785a-8f1f-11e7-876d-042da840e146.png">
<img width="965" alt="dashboard" src="https://user-images.githubusercontent.com/1282577/29971295-a85e1c5e-8f1f-11e7-8e12-5234705ca73c.png">

Reply:
Surprised it's that much slower in IE11!